frontend/src/Settings/Tags/Tag.js

```diff
@@ -97,6 +99,13 @@ class Tag extends Component {
                   </div>
               }
 
+              {
+                !!importListIds.length &&
```

Comment:
Nit: Convert to ternary and can drop `!!`

Reply:
I'll go ahead and change the other instances in this file as well